fsd_utils/services/aws.py

```diff
@@ -0,0 +1,347 @@
+import json
+from datetime import datetime
+
+import boto3
+from botocore.exceptions import ClientError
+
+
+class SQSClient:
+    def __init__(
+        self,
+        aws_access_key_id=None,
+        aws_secret_access_key=None,
+        region_name="us-west-1",
+        endpoint_url=None,
+        **kwargs,
+    ):
+        if aws_access_key_id and aws_access_key_id:
```

Comment:
Should this be checking for two different variables?

Reply:
yes. good catch